src/v2/view-builder/views/duo/ChallengeDuoAuthenticatorView.js

```diff
@@ -0,0 +1,44 @@
+import { loc, createCallout } from 'okta';
+import Duo from 'duo';
+import BaseForm from '../../internals/BaseForm';
+import BaseAuthenticatorView from '../../components/BaseAuthenticatorView';
+import AuthenticatorEnrollFooter from '../../components/AuthenticatorEnrollFooter';
+
+const Body = BaseForm.extend({
+
+  noButtonBar: true,
+
+  title  () {
```

Comment:
style-nit: Extra spaces

```suggestion
  title () {
```